data/coverage/2021.yml

```diff
@@ -114,6 +114,9 @@ november:
     - type: video
       title: LRUG November 2021 - Christian Gregg - Failing better with Load Shedding & Deadline Propagation across services
       url: https://assets.lrug.org/videos/2021/november/christian-gregg-failing-better-with-load-shedding-and-deadline-propagation-across-services-lrug-nov-2021.mp4
+    - type: slides
+      title: Failing better with Load Shedding & Deadline Propagation across services
+      url: https://github.com/lrug/lrug.org/files/7529520/presentation.pdf
```

Comment:
Hi @CGA1123 - thanks for adding the slides.  One thing though, I don't understand how the PDF lives on a URL for this repo, without it being an uploaded file in the commit.  I'm glad it's not, as I don't want this repo to host lots of big binary files, but I'm curious about it.  I guess there's some part of the GitHub API that I'm not aware of, but it's defeating me to find a reference to it when googling for GitHub files because all the hits I'm finding are about files _in_ the git repo itself, and this feels like it's a file that's attached to, but not part of, the repo.  What am I missing?

Reply:
Hey, sorry for the confusion! I was also a bit surprised when I first saw the URL path.

This is a link to a file as uploaded within comment for example.
It isn't part of the `.git` data, but separate blob storage related to the repo, in the same way that an image uploaded in a comment or PR description might be.

Depending on the type of file it seems that the URL returned after uploading differs it seems, likely something to do with whether it's routed via a CDN or not/whether it is renderable on load or requires explicit download 🤷‍♂️ 

e.g.

A .pdf file:
[presentation.pdf](https://github.com/lrug/lrug.org/files/7530959/presentation.pdf)

A .log file:
[hello.log](https://github.com/lrug/lrug.org/files/7530992/hello.log)

A .png file:
<img width="1299" alt="chart" src="https://user-images.githubusercontent.com/7707413/141595933-0cee9bed-a496-40c1-b5de-38eea52aaaa1.png">


---

I'm happy to host the file itself elsewhere if that is preferred? Or also happy for it to be hosted next to the video files if that's easier! 😄

Reply:
Thanks for the explanation @CGA1123.  I'm just confused, because until you wrote this comment I couldn't see where the file had been uploaded - like I'd have expected it to be on the PR description or something?  Either way, I think this is fine, so I'll merge.  Thanks for the PR (even if it took me a while to understand it), and thanks again for the talk.